spec/rubocop/cop/style/mutable_constant_spec.rb

```diff
@@ -157,6 +157,123 @@
           RUBY
         end
       end
+
+      context 'when using shareable_constant_value: literal' do
+        let(:prefix) { '# shareable_constant_value: literal' }
+
+        it_behaves_like 'immutable objects', '[1, 2, 3]'
+        it_behaves_like 'immutable objects', '%w(a b c)'
+        it_behaves_like 'immutable objects', '{ a: 1, b: 2 }'
+        it_behaves_like 'immutable objects', "'str'"
+        it_behaves_like 'immutable objects', '"top#{1 + 2}"'
+        it_behaves_like 'immutable objects', '1'
+        it_behaves_like 'immutable objects', '1.5'
+        it_behaves_like 'immutable objects', ':sym'
+        it_behaves_like 'immutable objects', 'FOO + BAR'
+        it_behaves_like 'immutable objects', 'FOO - BAR'
+        it_behaves_like 'immutable objects', "'foo' + 'bar'"
+        it_behaves_like 'immutable objects', "ENV['foo']"
+        it_behaves_like 'immutable objects', "::ENV['foo']"
```

Comment:
Could these be grouped together and reused instead of copied?

Reply:
Thanks for the comment. Refactored.